c10/util/Half.h

```diff
@@ -18,10 +18,8 @@
 
 #if defined(__cplusplus) && (__cplusplus >= 201103L)
 #include <cmath>
-#include <cstdint>
```

Comment:
I think removing these did cause a compile error that patched in a different PR.